timer.go

```diff
@@ -0,0 +1,42 @@
+package libtime
+
+import (
+	"time"
+)
+
+// StopFunc is used to stop a time.Timer.
+type StopFunc func()
```

Comment:
`time.Timer.Stop`'s godoc describes how calling stop affects the timer's channel.

It may make sense to change the signature of `StopFunc` to `func() bool` or document its behavior.

I guess it'd make sense to mention how calling the StopFunc relates to the Timer's Stop function, too.